core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -45,7 +45,7 @@ class CivilizationInfo {
     @Transient var viewableInvisibleUnitsTiles = setOf<TileInfo>()
 
     /** Contains cities from ALL civilizations connected by trade routes to the capital */
-    @Transient var citiesConnectedToCapital = listOf<CityInfo>()
+    @Transient var citiesConnectedToCapital = mapOf<CityInfo, Set<String>>()
```

Comment:
Since we're adding in a set of values, the name should be changed to reflect this - 'ToMediums' is what I used in the function but there's probably a better name

Reply:
i added postfix, couldn't come up with anything shorter